lib/recognize-stream.ts

```diff
@@ -92,12 +51,12 @@ class RecognizeStream extends Duplex {
     return contentType.fromHeader(buffer);
   }
 
-  private options: Options;
+  private options: RecognizeStream.Options;
   private authenticator: Authenticator;
   private listening: boolean;
   private initialized: boolean;
   private finished: boolean;
-  private socket;
+  private socket: w3cWebSocket;
```

Comment:
Huh, I thought I added this before and got compiler issues. I guess I was doing something different. Glad to see this 👍